agent/conf/agent.properties

```diff
@@ -398,3 +395,6 @@ iscsi.session.cleanup.enabled=false
 #consoleproxy.httpListenPort=443
 
 #ping.retries=5
+
+# The number of iothreads. There should be only 1 or 2 IOThreads per VM CPU (default is 1)
```

Comment:
Maybe add a comment here that a value >1 is rarely used and people shouldn't touch this? IIRC there is no reason to go higher then 1.